apps/ios/src/index.tsx

```diff
@@ -1,15 +1,17 @@
 'use strict';
 
 import * as React from 'react';
-import { AppRegistry } from 'react-native';
+import { AppRegistry, SafeAreaView } from 'react-native';
 import { ThemeProvider } from '@uifabricshared/theming-react-native';
 import { FabricTester, IFabricTesterProps, customRegistry } from '@fluentui-react-native/tester';
 
 const FluentTester: React.FunctionComponent<IFabricTesterProps> = props => {
   return (
-    <ThemeProvider registry={customRegistry}>
-      <FabricTester {...props} />
-    </ThemeProvider>
+    <SafeAreaView>
```

Comment:
Thanks for adding this!

Reply:
I'll extract this into a separate PR since it's useful regardless.

Reply:
Here: #303. I ran `yarn change` but it didn't do anything. Is that normal?

Reply:
Yea that's fine. We don't verify change files like how you would if you're using rush in, say, sdx-platform. I don't run yarn change for any changes in this repository

Reply:
Your [README](https://github.com/microsoft/fluentui-react-native#beachball) suggests otherwise :stuck_out_tongue:

Reply:
Ah yes sorry for the confusion- that applies to Windows. The Apple documentation is lacking right now